spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -925,6 +925,14 @@ public void setCommands(List<String> commands) {
             this.commands = commands;
         }
 
+        public List<String> getArgs() {
```

Comment:
It seems like before this change we were using commands to contain both command + args. If users were previously doing `commands = "sh,-c,script2.sh"` will they now have to use `commands = "sh" + args = "-c,script2.h"` or will both variants still be supported? IOW - will this break existing configurations?

Reply:
This test was just to validate the provision of args are applied as expected.

`commands` overrides the entrypoint.
`args` is added to the entrypoint.